test/built-ins/TypedArray/prototype/Symbol.toStringTag/this-is-not-object-strict-mode.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.3.31
+description: Return undefined when `this` is null or undefined
+info: >
+  22.2.3.31 get %TypedArray%.prototype [ @@toStringTag ]
+
+  1. Let O be the this value.
+  2. If Type(O) is not Object, return undefined.
+  ...
+includes: [testTypedArray.js]
+flags: [onlyStrict]
```

Comment:
Actually, that's only true for ECMAScript function objects. For built-in function objects like the ones under test, OrdinaryCallBindThis is *not* used. This means that both inside strict mode and outside strict mode, when either `null` or `undefined` is specified as a "this" value, that is the actual value during function execution (not the global object).